pom.xml

```diff
@@ -1,34 +1,127 @@
-	<version>0.1</version>
-	<name>api-service-e-signing</name>
-	<properties>
-		<!-- Pipeline properties -->
-		<docker.image.name>ms-e-signing</docker.image.name>
-		<published.container.port.test>8710</published.container.port.test>
-		<published.container.port.production>8710</published.container.port.production>
-	</properties>
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-devtools</artifactId>
-			<scope>runtime</scope>
-			<optional>true</optional>
-		</dependency>
-		<!-- Test -->
-		<dependency>
-			<groupId>se.sundsvall.dept44</groupId>
-			<artifactId>dept44-starter-test</artifactId>
-			<scope>test</scope>
-		</dependency>
-	</dependencies>
+<project
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>se.sundsvall.dept44</groupId>
+        <artifactId>dept44-service-parent</artifactId>
+        <version>4.3.1</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>se.sundsvall</groupId>
+    <artifactId>api-service-e-signing</artifactId>
+    <version>1.0</version>
+    <name>api-service-e-signing</name>
+    <properties>
+        <generated-sources-path>${project.build.directory}/generated-sources
+        </generated-sources-path>
+        <generated-sources-java-path>src/main/java</generated-sources-java-path>
+        <!-- Pipeline properties -->
+        <docker.image.name>ms-e-signing</docker.image.name>
+        <published.container.port.test>8710</published.container.port.test>
+        <published.container.port.production>8710</published.container.port.production>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-devtools</artifactId>
+            <scope>runtime</scope>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>se.sundsvall.dept44</groupId>
+            <artifactId>dept44-common-validators</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>se.sundsvall.dept44</groupId>
+            <artifactId>dept44-starter-feign</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <!-- Test -->
+        <dependency>
+            <groupId>se.sundsvall.dept44</groupId>
+            <artifactId>dept44-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Does the service really need 1024 mb memory? If it can execute with the default memory setting, remove
``` 
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-surefire-plugin</artifactId>
                <configuration>
                    <argLine>${surefireArgLine} -Xmx1024m</argLine>
                </configuration>
            </plugin>
```
?